src/Bicep.Cli/Commands/BuildParamsCommand.cs

```diff
@@ -54,10 +67,49 @@ public async Task<int> RunAsync(BuildParamsArguments args)
                 return 1;
             }
 
+            var paramsOverrides = Environment.GetEnvironmentVariable("BICEP_PARAMETERS_OVERRIDES");
+
+            if (paramsOverrides is { })
+            {
+                var paramsOverridesJson = JObject.Parse(paramsOverrides);
+
+                var fileContents = await File.ReadAllTextAsync(paramsInputPath);
+                var sourceFile = SourceFileFactory.CreateBicepParamFile(PathHelper.FilePathToFileUrl(paramsInputPath), fileContents);
+
+                var newProgramSyntax = CallbackRewriter.Rewrite(sourceFile.ProgramSyntax, syntax =>
+                {
+                    if (syntax is not ParameterAssignmentSyntax paramSyntax)
+                    {
+                        return syntax;
+                    }
+
+                    var overrideJsonValue = paramsOverridesJson.GetValue(paramSyntax.Name.IdentifierName);
```

Comment:
Looks like `GetValue` will return `null` both if the parameter is not present, or is present with a `null` value. Should we allow users to set values to null?

Reply:
In that case with current implementation, we will skip the override if the user actually intended to provide a `null` value. 

Values in `.bicepparam` file take precedence over defaults in the `.bicep` file and if a `null` is given in `.bicepparam` the default is used I think. If the user wants to do a deployment where they do not want to use the `.bicepparam` value for a specific parameter and use the default instead, they can provide inline `null` value.  How common do we think this is?

We could also just leave it as is (i.e: ignore `null` overrides) and enable later if users request

Reply:
> We could also just leave it as is (i.e: ignore null overrides) and enable later if users request

The problem with this (and the reason I'm calling it out now) is that this would be a breaking change. Feels like it would be worth deciding upon the expected behavior now to avoid this possibility.